events/tests/test_event_get.py

```diff
@@ -1113,6 +1113,44 @@ def test_keywordset_search(
     )
 
 
+@pytest.mark.django_db
+def test_keywordset_search_match_audience(
+    api_client,
+    event,
+    event2,
+    event3,
+    keyword,
+    keyword2,
+    keyword3,
+    keyword_set,
+    keyword_set2,
+):
+    # Same as previous but test match to audience.
```

Comment:
While true at the present, the "previous" can change in the future. This comment probably won't age well and if I ever saw this in the wild, I probably wouldn't trust this comment. Just leave this out or describe the test in a way that doesn't rely on other tests.